src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -230,6 +231,24 @@ public bool IsBackgroundGridVisible
             }
         }
 
+        /// <summary>
+        /// Indicates the default factor read from DynamoSettings.xml and can be updated using the Preferences panel
```

Comment:
Since this is a comment for public property, let's say `default geometry scale factor for new workspace`

Reply:
This comment was updated in the next commit:
https://github.com/DynamoDS/Dynamo/pull/13656/commits/789f51c4a3c106156940924585e1ca40abd9842b